lib/grplatt.gi

```diff
@@ -1900,7 +1900,7 @@ local G,	# group
 	  Info(InfoLattice,2,"factorsize=",Index(j,N),"/",Index(M,N));
 
 	  # reasons not to go complements
-	  if (HasAbelianFactorGroup(j,N) and
+	  if (HasElementaryAbelianFactorGroup(j,N) and
```

Comment:
I think I understand the new code below; but I don't understand what the change in this line is about. Could you briefly explain?

Reply:
The code (at some point) calculates complements. If this is in an abelian group it can be hard (which is why the code used to delegate this to `SubgroupsSolvableGroup`, finding invariant subgroups. In the meantime the code got changed to use action on the cohomology, so the old diversion to subgroups code is not worth doing any longer (generically), but only the special case diversion for the case of modules. (I.e. this change should have happened long ago, but I did not notice the performance difference.)